sched/semaphore/sem_clockwait.c

```diff
@@ -173,7 +179,11 @@ int nxsem_clockwait(FAR sem_t *sem, clockid_t clockid,
 
   wd_cancel(&rtcb->waitdog);
 
-out:
+  /* We can now restore interrupts and delete the watchdog */
+
+success_with_irqdisabled:
+errout_with_irqdisabled:
```

Comment:
What is the benefit of this vs single `out:`?

Reply:
here is redundant, i will fix this